src/useReducer.ts

```diff
@@ -66,14 +66,16 @@ export function useHookedReducer<S, A extends Action<any>>(
 export function useReducer<S, A extends Action<any> = Action<any>>(
     reducer: Reducer<S, A>,
     initialState: S,
+    init: (state: S) => S = state => state,
     id: string | number
 ): [S, Dispatch<A>] {
+    const inspectorStore = useContext(StateInspectorContext)
     const [store, reducerId] = useMemo<[EnhancedStore, string | number]>(
-        () => [useContext(StateInspectorContext), id],
+        () => [inspectorStore, id],
```

Comment:
So the previous version is no longer working because it is no longer possible to use a hook inside `useMemo`?

Reply:
Well, first thing is API change with new init param to useReducer.

But seconds is this. I've just tested it just to be sure I'm not misleading you :)

```sh
// console.error
index.js:1446 Warning: Do not call Hooks inside useEffect(...), useMemo(...), or other built-in Hooks. You can only call Hooks at the top level of your React function. For more information, see https://fb.me/rules-of-hooks
 
// following crash
Should have a queue. This is likely a bug in React. Please file an issue.
▼ 7 stack frames were expanded.
invariant
node_modules/react-dom/cjs/react-dom.development.js:57
updateReducer
node_modules/react-dom/cjs/react-dom.development.js:13468
updateState
node_modules/react-dom/cjs/react-dom.development.js:13618
useState
node_modules/react-dom/cjs/react-dom.development.js:14098
useState
node_modules/react/cjs/react.development.js:1495
useHookedReducer
node_modules/reinspect/dist/index.es.js:110
useReducer$1
```

Reply:
waiting for this merge